package.json

```
@@ -38,7 +38,7 @@
     "@types/node": "^12.12.17",
     "jest": "^24.9.0",
     "ts-jest": "^24.2.0",
-    "typescript": "^3.7.3"
+    "typescript": "^3.9.7"
```

Is this typescript bump strictly necessary anymore?

```suggestion
    "typescript": "^3.7.3"
```

---

Not sure, maybe not ;)